os/arch/arm/src/armv7-m/mpu.h

```diff
@@ -160,6 +160,12 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define STACK_GUARD_ADDR_MASK	0xffffffe0	/* Bits N-31:  Used for MPU Region base addrese */
```

Comment:
STACK_GUARD_ADDR_MASK is not used anywhere. Please remove it.

Reply:
changed